rtv/__main__.py

```diff
@@ -47,6 +46,8 @@ def load_config():
     if 'unicode' in defaults:
         defaults['unicode'] = config.getboolean('rtv', 'unicode')
 
+    rtvconfig.theme = defaults['theme']
```

Comment:
We should make sure `"theme"` is in `defaults`, otherwise we'll get a `KeyError`.

```python
if 'theme' in defaults:
    rtvconfig.theme = defaults['theme']
else:
    rtvconfig.theme = None
```